pages/core-unit/[code]/index.tsx

```diff
@@ -1,11 +1,34 @@
-import { NextPage } from 'next';
+import { NextPage, GetServerSideProps, InferGetServerSidePropsType, GetServerSidePropsContext } from 'next';
 import React from 'react';
+import _ from 'lodash';
+
 import CuAboutContainer from '../../../src/stories/containers/cu-about/cu-about-container';
+import { CuAbout, fetchCoreUnitByCode } from '../../../src/stories/containers/cu-about/cu-about.api';
 
-const CoreUnitAboutPage:NextPage = () => {
+const CoreUnitAboutPage: NextPage = ({ code, cuAbout, contributors }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   return (
-    <CuAboutContainer />
+    <CuAboutContainer code={code} cuAbout={cuAbout as CuAbout} contributors={contributors} />
   );
 };
-
 export default CoreUnitAboutPage;
+export const getServerSideProps: GetServerSideProps = async(context: GetServerSidePropsContext) => {
+  const { query } = context;
+  const code = query.code as string;
+  const cuAbout = await fetchCoreUnitByCode(code);
+  const contributorCommitment = cuAbout.contributorCommitment;
+
+  console.log('cuAbout', cuAbout);
```

Comment:
You forgot to remove the `console.log`. Also, there is no need to check if the `contributorCommitment` length is 0 since if `cuAbout` is empty, `contributorCommitment` will be empty too

Reply:
you are right i will remove both of them